Src/LinearSolvers/MLMG/AMReX_MLMG.cpp

```diff
@@ -1438,25 +1462,29 @@ MLMG::apply (const Vector<MultiFab*>& out, const Vector<MultiFab*>& a_in)
     Vector<MultiFab*> in(namrlevs);
     Vector<MultiFab> in_raii(namrlevs);
     Vector<MultiFab> rh(namrlevs);
+    int nghost = 0;
+    if (cf_strategy == CFStrategy::ghostnodes) nghost = linop.getNGrow();
 
     for (int alev = 0; alev < namrlevs; ++alev)
     {
-        if (a_in[alev]->nGrow() == 1)
+        if (a_in[alev]->nGrow() >= 1)
```

Comment:
Let's not change the current behavior for other codes.

Reply:
Done (see previous comment)